### PR TITLE
[type: feature] add http file to examples[grpc,http,spring cloud]

### PR DESCRIPTION
**Pull request by li-keguo**
add http file to examples[grpc,http,spring cloud]  [#2044]

other examples project learning, subsequent submission

work list:

- [ ] add http file to examples[grpc,http,spring cloud]  
- [ ] remove HttpServerConfig#environment [shengyu-examples-http], it never used
- [ ]  add ShenyuWebHandler#ShenyuWebHandler()[shengyu-web] java docs, it build warning

### CODE REVIEW

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/JwtUtils.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ public static UserInfo getUserInfo() {
      */
     public static String getIssuer(final String token) {
         DecodedJWT jwt = JWT.decode(token);
-        return Optional.ofNullable(jwt).map(item -> item.getClaim("userName").asString()).orElse("");
+        return Optional.of(jwt).map(item -> item.getClaim("userName").asString()).orElse("");
```
Comment (yu199195) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/JwtUtils.java:61]: why modify Optional.ofNullable to Optional.of?   jwt won't it be Null？

File: shenyu-examples/shenyu-examples-grpc/src/main/http/grpc-test-api.http [hunk 2]
```diff
@@ -0,0 +1,70 @@
+
```
Comment (yu199195) [inline shenyu-examples/shenyu-examples-grpc/src/main/http/grpc-test-api.http:1]: add apache 2.0 license

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/JwtUtils.java (hunk 1 above)
Comment (li-keguo) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/JwtUtils.java:61]: JWT#decode(String) implement is:
`
new JWTDecoder(parser, token)
`
it not null

File: shenyu-examples/shenyu-examples-grpc/src/main/http/grpc-test-api.http (hunk 2 above)
Comment (li-keguo) [reply shenyu-examples/shenyu-examples-grpc/src/main/http/grpc-test-api.http:1]: ok

**Comment by yu199195** [comment]
Hi, can you  exec  mvn clean install  Before the Pull Request？

**Comment by li-keguo** [comment]
> Hi, can you exec mvn clean install Before the Pull Request？

yes, exec it in local

**Comment by li-keguo** [comment]
I don't know why the build failed，
I read build log
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:testCompile (default-testCompile) on project shenyu-admin: Compilation failure
Error:  /home/runner/work/incubator-shenyu/incubator-shenyu/shenyu-admin/src/test/java/org/apache/shenyu/admin/AbstractReflectGetterSetterTest.java:[115,55] cannot access com.sun.beans.introspect.PropertyInfo
```
I didn't modify these

**Pull request closed without merging**